manifests/resource/server.pp

```diff
@@ -592,8 +594,29 @@
   if $ssl {
     # Access and error logs are named differently in ssl template
 
-    File <| title == $ssl_cert or path == $ssl_cert or title == $ssl_key or path == $ssl_key |>
-    -> concat::fragment { "${name_sanitized}-ssl-header":
+    if $ssl_key.is_a(Array) {
```

Comment:
Should we prefer the built-in `=~` operator these days over the stdlib `is_a` function?